### PR TITLE
Mariocube.EasyCSM version 1.4

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103189)

### CODE REVIEW

File: manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4
```
Comment (mdanish-kh) [inline manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.installer.yaml:5]:
```suggestion
PackageVersion: 1.4
AppsAndFeaturesEntries:
  - DisplayVersion:
```
The installer doesn't write a display version string to registry

File: manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4
+Installers:
+- InstallerUrl: https://repo.mariocube.com/Wii/PC%20Applications/Easy%20CSM/ECSM%201.4.exe
+  Architecture: x86
+  InstallerType: inno
```
Comment (mdanish-kh) [inline manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.installer.yaml:9]:
```suggestion
  InstallerType: inno
  Scope: machine
  ProductCode: '{C8AB2AA8-501D-4553-B096-CD78EFB90366}_is1'
```
For better package matching

File: manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4
+PackageLocale: en-US
+Publisher: Mariocube
+PackageName: Easy CSAM
```
Comment (mdanish-kh) [inline manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.locale.en-US.yaml:8]:
```suggestion
PackageName: ECSM
```
Must match whatever is written to registry (shown in Apps&Features or `appwiz.cpl`)

File: manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4
+PackageLocale: en-US
+Publisher: Mariocube
```
Comment (mdanish-kh) [inline manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.locale.en-US.yaml:7]:
```suggestion
Publisher: c0mputeRb0y
```
Must match whatever is written to registry (shown in Apps&Features or `appwiz.cpl`)

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**